.github/workflows/update_google_font.yml

```diff
@@ -0,0 +1,16 @@
+name: 'Update Google Fonts'
+
+on:
+  schedule:
+    # This runs once a week: https://crontab.guru/#0_0_*_*_0
+    - cron: '0 0 * * 0'
+  workflow_dispatch:
+
+jobs:
+  update-fonts:
+    name: update fonts
+    uses: vercel/next.js/.github/workflows/build_reusable.yml@canary
+    with:
+      skipForDocsOnly: 'yes'
+      skipInstallBuild: 'yes'
+      afterBuild: pnpm install && pnpm update-google-fonts
```

Comment:
Are you missing a `git commit && git push`? 

I took a quick look at `update-google-fonts` and it looks like it just writes files so this job won't do anything.